json/access/src/test/java/ch/qos/logback/contrib/json/access/JsonLayoutTest.java

```diff
@@ -146,19 +147,19 @@ public void jsonLayout() throws Exception {
         jsonLayout.setContext(context);
         String log = jsonLayout.doLayout(iAccessEvent);
 
-        assertTrue(log.contains(String.format("%s=%s", JsonLayout.REMOTEHOST_ATTR_NAME, event.getRemoteHost())));
-        assertTrue(log.contains(String.format("%s=%s", JsonLayout.REMOTEUSER_ATTR_NAME, event.getRemoteUser())));
-        assertTrue(log.contains(String.format("%s=%s", JsonLayout.REMOTEADDR_ATTR_NAME, event.getRemoteAddr())));
-        assertTrue(log.contains(String.format("%s=%s", JsonLayout.METHOD_ATTR_NAME, event.getMethod())));
-        if(event.getRequestHeaderMap().size() == 2){
+        assertThat(log.contains(String.format("%s=%s", JsonLayout.REMOTEHOST_ATTR_NAME, event.getRemoteHost())), is(true));
```

Comment:
This could be improved with:

```java
// import static org.junit.matchers.JUnitMatchers.containsString;
assertThat(log, containsString(...));
```

which would output a helpful error like this:

```
java.lang.AssertionError: 
Expected: a string containing "level=INFO1"
     got: "{timestamp=1464978122990, level=INFO, thread=main, logger=STR_LIST, message=Info message}"

Expected :a string containing "level=INFO1"
Actual   :"{timestamp=1464978122990, level=INFO, thread=main, logger=STR_LIST, message=Info message}"
```

Compare that to:

```
java.lang.AssertionError: 
Expected: is <true>
     got: <false>

Expected :is <true>
Actual   :<false>
```